pacnic in example: Q.db does not set

# Problem

I'm not sure if it's my own problem. 

Running `example` directly reports panic at L14 because the `Q.db` is `nil` pointer.
```bash
/gen/examples$ go run main.go
hello world
bye~
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5b19f8]

goroutine 1 [running]:
gorm.io/gorm.(*DB).Session(0x0, 0xc0000c7230)
        /home/bige/go/pkg/mod/gorm.io/gorm@v1.23.5/gorm.go:215 +0x38
gorm.io/gorm.(*DB).WithContext(...)
        /home/bige/go/pkg/mod/gorm.io/gorm@v1.23.5/gorm.go:298
gorm.io/gen.(*DO).WithContext(0xc0001fc400?, {0xa64388?, 0xc000024090?})
        /home/bige/gen/do.go:151 +0x85
gorm.io/gen/examples/dal/query.mytableDo.WithContext(...)
        /home/bige/gen/examples/dal/query/mytables.gen.go:92
gorm.io/gen/examples/dal/query.(*mytable).WithContext(0xc0000c7cd8?, {0xa64388?, 0xc000024090?})
        /home/bige/gen/examples/dal/query/mytables.gen.go:63 +0x8b
gorm.io/gen/examples/biz.Query({0xa615c0?, 0xc000010018?})
        /home/bige/gen/examples/biz/query.go:14 +0x78
main.main()
        /home/bige/gen/examples/main.go:21 +0xbc
exit status 2
```

# Solution

Set the default database for `query` at `examples/main.go#L20`

```go
query.SetDefault(dal.DB)
```

I will create a pull request soon for this.

--- a/examples/main.go
+++ b/examples/main.go
@@ -18,5 +18,6 @@ func main() {
 	fmt.Println("hello world")
 	defer fmt.Println("bye~")
 
+	query.SetDefault(dal.DB)
 	biz.Query(context.Background())
 }

